tests/test_views.py

```diff
@@ -87,3 +122,38 @@ def test_generate_password_view(client):
 
     assert res.status_code == 200
     assert res.context["form"].initial["value"]
+
+
+@pytest.mark.django_db
+def test_view_secret_more_then_once(client):
+    secret, url_part = create_secret(
+        "My secret value", expires_in=ONE_HOUR, view_once=False
+    )
+
+    res = client.get(
+        reverse("django_secret_sharing:retrieve", kwargs={"url_part": url_part})
+    )
+
+    res2 = client.get(
+        reverse("django_secret_sharing:retrieve", kwargs={"url_part": url_part})
+    )
+
+    assert res.status_code == 200
+    assert res2.status_code == 200
+
+
+@pytest.mark.django_db
+def test_required_form_fields(client):
+    no_expires_res = client.post(
+        reverse("django_secret_sharing:create"),
+        data={"value": "My secret value", "expires": "", "view_once": True},
+    )
+
+    no_value_res = client.post(
+        reverse("django_secret_sharing:create"),
+        data={"value": "", "expires": ONE_HOUR, "view_once": True},
```

Comment:
hoezo geeft dit een error? 1 hour is geldige waarde toch?
hier expliciet een waarde geven die NIET in de lijst zit (nog beter is dit in een aparte test een expliciete naam geven zodat het duideljjker is wat er wordt getest)